src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Callback.cs

```diff
@@ -18,11 +20,13 @@ internal partial class NavigateToItemProvider
     {
         private class NavigateToItemProviderCallback : INavigateToSearchCallback
         {
+            private readonly Solution _solution;
             private readonly INavigateToItemDisplayFactory _displayFactory;
             private readonly INavigateToCallback _callback;
 
-            public NavigateToItemProviderCallback(INavigateToItemDisplayFactory displayFactory, INavigateToCallback callback)
+            public NavigateToItemProviderCallback(Solution solution, INavigateToItemDisplayFactory displayFactory, INavigateToCallback callback)
```

Comment:
part of the ystem used to involve a callback that would pass along a project+item.  The former is redundant given that everyone starts with a solution snapshot, and a navto item always contains a project-id in it anyways.